docs/bonus_guides/custom_primary_key.md

```diff
@@ -13,20 +13,24 @@ Let's say that we need a JSON resource that stores rows of team athletes. Each a
 We can generate that resource like this.
 
 ```console
-$ mix phoenix.gen.json Player players name:string position:string number:integer
-* creating priv/repo/migrations/20150908003815_create_player.exs
-* creating web/models/player.ex
-* creating test/models/player_test.exs
-* creating web/controllers/player_controller.ex
-* creating web/views/player_view.ex
-* creating test/controllers/player_controller_test.exs
-* creating web/views/changeset_view.ex
+$ mix phx.gen.json Rosters Player players name:string position:string number:integer
+* creating lib/hello/controllers/player_controller.ex
+* creating lib/hello/views/player_view.ex
+* creating test/hello/controllers/player_controller_test.exs
+* creating lib/hello/views/changeset_view.ex
+* creating lib/hello/controllers/fallback_controller.ex
+* creating lib/hello/rosters/player.ex
+* creating priv/repo/migrations/20170906132454_create_players.exs
+* creating lib/hello/rosters/rosters.ex
+* injecting lib/hello/rosters/rosters.ex
+* creating test/hello/rosters/rosters_test.exs
+* injecting test/hello/rosters/rosters_test.exs
 
-Add the resource to your api scope in web/router.ex:
+Add the resource to your :api scope in lib/hello/router.ex:
```

Comment:
I don't think the symbol notation is necessary. BUT if you do have strong feelings for :api, put it in code blocks `:api`

Reply:
@jeregrine This is an output of above `mix phx.gen.json ... ` command.